sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Integration.Tests/DistroWebAppLiveTests.cs

```diff
@@ -89,11 +91,13 @@ public async Task VerifyDistro()
             Assert.True(res.Equals("Response from Test Server"), "If this assert fails, the in-process test server is not running.");
 
             // SHUTDOWN
+            var tracerProvider = app.Services.GetRequiredService<TracerProvider>();
+            tracerProvider.ForceFlush();
+            tracerProvider.Shutdown();
 
-            // NOTE: If this test starts failing, Flushing may be necessary.
-            //var tracerProvider = app.Services.GetRequiredService<TracerProvider>();
-            //tracerProvider.ForceFlush();
-            //tracerProvider.Shutdown();
+            var meterProvider = app.Services.GetRequiredService<MeterProvider>();
+            meterProvider.ForceFlush();
+            meterProvider.Shutdown();
```

Comment:
why do we need to this explicitly? Shutting down the host will dispose the providers, thereby effecting shutdown effects.